docs/about/index.rst

```diff
@@ -17,13 +40,156 @@ Code of conduct
 This project follows the
 `GitHub community guidelines <https://help.github.com/en/github/site-policy/github-community-guidelines>`_.
 
+.. _history:
+
+Project history
+---------------
+
+Fairlearn was started in 2018 by Miro Dudik from Microsoft Research as a
+Python package to accompany the research paper,
+`A Reductions Approach to Fair Classification <http://proceedings.mlr.press/v80/agarwal18a/agarwal18a.pdf>`_.
+The package provided a reduction algorithm for mitigating unfairness in binary
+classification models&mdash;a setting that was commonly studied in the machine
+learning community.  The paper and the Python package were well received, so
+Miro Dudik and Hanna Wallach wanted to translate the research into an industry
+context.  However, they discovered that practitioners typically need to
+address more fundamental fairness issues before they can use specific
+algorithms, and that mitigating unfairness in binary classification models is
+a relatively rare use case. They also discovered that fairness assessment is a
+common need, along with access to domain-specific guides to fairness metrics
+and unfairness mitigation algorithms. Additionally, many use cases take the
+form of regression or ranking, rather than classification. As a result of
+these insights, fairness assessment and use-case notebooks became key
```

Comment:
"These insights" might work better as a bulleted list.

Reply:
@MiroDudik probably has some thoughts on this.

Reply:
I'd rather keep the history description more of an origin story in the form of a narrative.